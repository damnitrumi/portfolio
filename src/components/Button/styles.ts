import styled, { css } from "styled-components";

export const Wrapper = styled.button`
    ${() => css`
      border: none;
      background-color: transparent;
      cursor: pointer;
    `}
`;
