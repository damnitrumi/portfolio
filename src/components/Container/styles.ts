import styled, { css } from "styled-components";

export const Wrapper = styled.div`
    ${({ theme }) => css`
        background-color: ${theme.colors.primaryColor};
        padding: ${theme.paddings.containerDefault};
        height: 100vh;

        @media ${theme.media.ltaMedium}{
            padding: ${theme.paddings.containerLtaMedium};
        }
    `}
`;
