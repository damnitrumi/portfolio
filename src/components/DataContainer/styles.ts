import styled, { css } from "styled-components";

export const Wrapper = styled.div`
    ${() => css`
        max-width: 1100px;
        margin: 0 auto;
        height: 100%;
        padding: 200px 0;
    `}
`;
