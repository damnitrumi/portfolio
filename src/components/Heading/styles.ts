import styled, { css } from "styled-components";
import { HeadingProps } from ".";

const pickSize = (size: string) => {
  if (size) {
    return css`
      font-size: ${size};
    `;
  }
};

export const Title = styled.h1<HeadingProps>`
${({ size, as }) => css`
  color: white;
  ${pickSize(size)}
  font-weight: ${as == "h1" ? 500 : 400};
`}
`;
