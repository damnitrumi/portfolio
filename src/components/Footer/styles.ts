import styled, { css } from "styled-components";
import { Wrapper as Container } from "../Container/styles";

export const Wrapper = styled.div`
  ${() => css`
  `}
`;

export const FooterContainer = styled(Container)`
    background-color: transparent;
    height: auto;
    position: fixed;
    width: 100%;
    bottom: 0;
`;

export const LinksContainer = styled.div``;
