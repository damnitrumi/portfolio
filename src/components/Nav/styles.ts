import styled, { css } from "styled-components";
import { Wrapper as Container } from "../Container/styles";

export const Wrapper = styled.nav`
    ${() => css`
        display: flex;
        align-items: center;
        justify-content: space-between;
    `}
`;

export const imgContainer = styled.a`
    ${() => css`
        max-width: 150px;
    `}
`;

export const NavContainer = styled(Container)`
    background-color: transparent;
    height: auto;
    position: fixed;
    width: 100%;
    z-index: 1;

    @media ${({ theme }) => theme.media.ltaMedium}{
      background-color: ${({ theme }) => theme.colors.mediaGray};
    }
`;

export const bgMenu = styled.div`
    position: fixed;
    height: 100vh;
    width: 100%;
    top: 0;
    left: 0;
    background-color: rgba(0, 0, 0, 0.3);
    /* pointer-events: none; */
`;
