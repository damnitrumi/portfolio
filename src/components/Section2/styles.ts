import styled, { css } from "styled-components";
import { Wrapper as DataContainer } from "../DataContainer/styles";
import { Wrapper as Container } from "../Container/styles";
import { Wrapper as Text } from "../Text/styles";
import { Title as Heading } from "../Heading/styles";

export const Wrapper = styled.section`
    ${() => css`
      ${DataContainer}{
        justify-content: flex-start;

        & > ${Heading}{
          margin-bottom: 20px;
        };

        ${Text}:last-of-type{
          margin-top: 30px;
          margin-bottom: 30px;
        }

      }
      ${Container}{
        height: auto;
      }
    `}
`;

export const IconsContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    flex-wrap: wrap;
    align-items: center;
    justify-content: space-between;
    gap: 10px;
    margin-top: 10px;
    max-width: 400px;

    @media ${theme.media.ltaMedium}{
      gap: 2px;
      max-width: 100%;
    }
  `}

`;

export const IconContainer = styled.div`
      flex: 0 1 50px;
`;

export const HeadingLink = styled(Heading)`
  ${({ theme }) => css`

    a{
      background-color: ${theme.colors.white};
      padding: 15px 20px 10px 20px;
      max-width: max-content;
      border-radius: 5px;
      color: ${theme.colors.black};
      transition: all 200ms ease-in;
    }
    a:hover{
      color: ${theme.colors.white};
      background-color: ${theme.colors.hoverGreen};
    }

    @media ${theme.media.ltaMedium}{
      font-size: 1.6rem;
      a{
        max-width: 100%;
        text-align: center;
      }
    }

  `}

`;
