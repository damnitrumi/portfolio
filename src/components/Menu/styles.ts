import styled, { css } from "styled-components";
import { MenuBehaviorProps } from ".";
import { LinksContainer as IconsContainerMock } from "../Footer/styles";
import { StyledIconBase } from "@styled-icons/styled-icon";

export const Wrapper = styled.div<MenuBehaviorProps>`
    ${({ theme, menuVisible }) => css`
    position: fixed;
    width: 100%;
    max-width: 500px;
    top: 0;
    right: -100%;
    transition: all .8s ease-in-out;

    @media ${theme.media.ltaMedium} {
      max-width: 100%;
    };

    ${
      menuVisible &&
      css`
        right: 0;
    `
    }
    `}
`;

export const ButtonContainer = styled.div`
  display: flex;
  justify-content: flex-end;
`;
export const LinksContainer = styled.div`
  padding: 30px 0;

  a{
    padding: 20px 0;
    color: ${({ theme }) => theme.colors.menuGray};
    transition: all 200ms ease-in;
  }

  a:hover{
    color: ${({ theme }) => theme.colors.white};
  }
`;

export const IconsContainer = styled(IconsContainerMock)`
  ${StyledIconBase}{
    color: ${({ theme }) => theme.colors.menuGray};
    &:hover{
      color: ${({ theme }) => theme.colors.hoverGreen};
    }
  }
`;
