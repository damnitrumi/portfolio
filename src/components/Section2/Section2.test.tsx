import { screen } from "@testing-library/react";
import { renderTheme } from "../../styles/render-theme";
import { Section2 } from ".";

describe("< Section2/>", () => {
  it("should render Section2 and is components", () => {
    renderTheme(<Section2 />);

    expect(
      screen.getByText(
        "Desde maio de 2021 estudando para ser um desenvolvedor front-end.",
      ),
    ).toHaveStyleRule("font-size", "3.2rem");
    expect(
      screen.getByText("Atualmente trabalho com React.js/Next.JS."),
    ).toHaveStyleRule("font-size", "2.2rem");
    expect(screen.getByRole("link")).toHaveAttribute(
      "href",
      "https://github.com/damnitrumi?tab=repositories",
    );
  });

  it("should match snapshot", () => {
    const { container } = renderTheme(<Section2 />);

    expect(container).toMatchSnapshot();
  });
});
