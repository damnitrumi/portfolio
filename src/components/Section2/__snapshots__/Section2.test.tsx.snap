// Vitest Snapshot v1

exports[`< Section2/> > should match snapshot 1`] = `
.c2 {
  background-color: #1b1b1b;
  padding: 30px 60px;
  height: 100vh;
}

.c4 {
  max-width: 1100px;
  margin: 0 auto;
  height: 100%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
}

.c8 {
  color: #FFFFFF;
  font-size: 2.2rem;
  line-height: 30px;
}

.c6 {
  color: #FFFFFF;
  font-size: 3.2rem;
  font-weight: 400;
}

.c9 {
  color: #FFFFFF;
  font-size: 2.2rem;
  font-weight: 400;
}

.c0 .c3 {
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
}

.c0 .c3 > .c5 {
  margin-bottom: 20px;
}

.c0 .c3 .c7:last-of-type {
  margin-top: 30px;
  margin-bottom: 30px;
}

.c0 .c1 {
  height: auto;
}

.c11 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-wrap: wrap;
  -ms-flex-wrap: wrap;
  flex-wrap: wrap;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  gap: 10px;
  margin-top: 10px;
  max-width: 400px;
}

.c10 a {
  background-color: #FFFFFF;
  padding: 15px 20px 10px 20px;
  max-width: -webkit-max-content;
  max-width: -moz-max-content;
  max-width: max-content;
  border-radius: 5px;
  color: #000000;
  -webkit-transition: all 200ms ease-in;
  transition: all 200ms ease-in;
}

.c10 a:hover {
  color: #FFFFFF;
  background-color: #06c819;
}

.c12 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

@media (max-width:768px) {
  .c2 {
    padding: 20px;
  }
}

@media (max-width:768px) {
  .c11 {
    gap: 2px;
    max-width: 100%;
  }
}

@media (max-width:768px) {
  .c10 {
    font-size: 1.6rem;
  }

  .c10 a {
    max-width: 100%;
    text-align: center;
  }
}

<div>
  <section
    class="c0"
    id="sobre"
  >
    <div
      class="c1 c2"
    >
      <div
        class="c3 c4"
      >
        <h3
          class="c5 c6"
        >
          Desde maio de 2021 estudando para ser um desenvolvedor front-end.
        </h3>
        <p
          class="c7 c8"
        >
          Atualmente trabalho com React.js/Next.JS.
        </p>
        <p
          class="c7 c8"
        >
          No momento procuro a primeira oportunidade como desenvolvedor front-end júnior, e futuramente pretendo estudar sobre React Native pois penso em partir para o desenvolvimento Mobile.
        </p>
        <p
          class="c7 c8"
        >
          Inglês fluente.
        </p>
        <h3
          class="c5 c9 c10"
        >
          <a
            href="https://github.com/damnitrumi?tab=repositories"
            rel="noreferrer"
            target="_blank"
          >
            Clique aqui para ver os repositórios dos projetos
          </a>
        </h3>
        <div
          class="c11"
        >
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="50px"
            viewBox="0 0 24 24"
            width="50px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4.136 3.012h15.729l-1.431 16.15-6.451 1.826-6.414-1.826-1.433-16.15zm5.266 7.302-.173-2.035 7.533.002.173-1.963-9.87-.002.522 5.998h6.835l-.243 2.566-2.179.602-2.214-.605-.141-1.58H7.691l.247 3.123L12 17.506l4.028-1.08.558-6.111H9.402v-.001z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="50px"
            viewBox="0 0 24 24"
            width="50px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M4.192 3.143h15.615l-1.42 16.034-6.404 1.812-6.369-1.813L4.192 3.143zM16.9 6.424l-9.8-.002.158 1.949 7.529.002-.189 2.02H9.66l.179 1.913h4.597l-.272 2.62-2.164.598-2.197-.603-.141-1.569h-1.94l.216 2.867L12 17.484l3.995-1.137.905-9.923z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="50px"
            viewBox="0 0 24 24"
            width="50px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M3 3h18v18H3V3zm16.525 13.707c-.131-.821-.666-1.511-2.252-2.155-.552-.259-1.165-.438-1.349-.854-.068-.248-.078-.382-.034-.529.113-.484.687-.629 1.137-.495.293.09.563.315.732.676.775-.507.775-.507 1.316-.844-.203-.314-.304-.451-.439-.586-.473-.528-1.103-.798-2.126-.775l-.528.067c-.507.124-.991.395-1.283.754-.855.968-.608 2.655.427 3.354 1.023.765 2.521.933 2.712 1.653.18.878-.652 1.159-1.475 1.058-.607-.136-.945-.439-1.316-1.002l-1.372.788c.157.359.337.517.607.832 1.305 1.316 4.568 1.249 5.153-.754.021-.067.18-.528.056-1.237l.034.049zm-6.737-5.434h-1.686c0 1.453-.007 2.898-.007 4.354 0 .924.047 1.772-.104 2.033-.247.517-.886.451-1.175.359-.297-.146-.448-.349-.623-.641-.047-.078-.082-.146-.095-.146l-1.368.844c.229.473.563.879.994 1.137.641.383 1.502.507 2.404.305.588-.17 1.095-.519 1.358-1.059.384-.697.302-1.553.299-2.509.008-1.541 0-3.083 0-4.635l.003-.042z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="37px"
            viewBox="0 0 24 24"
            width="37px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M1.125 0C.502 0 0 .502 0 1.125v21.75C0 23.498.502 24 1.125 24h21.75c.623 0 1.125-.502 1.125-1.125V1.125C24 .502 23.498 0 22.875 0zm17.363 9.75c.612 0 1.154.037 1.627.111a6.38 6.38 0 0 1 1.306.34v2.458a3.95 3.95 0 0 0-.643-.361 5.093 5.093 0 0 0-.717-.26 5.453 5.453 0 0 0-1.426-.2c-.3 0-.573.028-.819.086a2.1 2.1 0 0 0-.623.242c-.17.104-.3.229-.393.374a.888.888 0 0 0-.14.49c0 .196.053.373.156.529.104.156.252.304.443.444s.423.276.696.41c.273.135.582.274.926.416.47.197.892.407 1.266.628.374.222.695.473.963.753.268.279.472.598.614.957.142.359.214.776.214 1.253 0 .657-.125 1.21-.373 1.656a3.033 3.033 0 0 1-1.012 1.085 4.38 4.38 0 0 1-1.487.596c-.566.12-1.163.18-1.79.18a9.916 9.916 0 0 1-1.84-.164 5.544 5.544 0 0 1-1.512-.493v-2.63a5.033 5.033 0 0 0 3.237 1.2c.333 0 .624-.03.872-.09.249-.06.456-.144.623-.25.166-.108.29-.234.373-.38a1.023 1.023 0 0 0-.074-1.089 2.12 2.12 0 0 0-.537-.5 5.597 5.597 0 0 0-.807-.444 27.72 27.72 0 0 0-1.007-.436c-.918-.383-1.602-.852-2.053-1.405-.45-.553-.676-1.222-.676-2.005 0-.614.123-1.141.369-1.582.246-.441.58-.804 1.004-1.089a4.494 4.494 0 0 1 1.47-.629 7.536 7.536 0 0 1 1.77-.201zm-15.113.188h9.563v2.166H9.506v9.646H6.789v-9.646H3.375z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="37px"
            viewBox="0 0 24 24"
            width="37px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <circle
              cx="12"
              cy="11.245"
              r="1.785"
            />
            <path
              d="m7.002 14.794-.395-.101c-2.934-.741-4.617-2.001-4.617-3.452 0-1.452 1.684-2.711 4.617-3.452l.395-.1.111.391a19.507 19.507 0 0 0 1.136 2.983l.085.178-.085.178c-.46.963-.841 1.961-1.136 2.985l-.111.39zm-.577-6.095c-2.229.628-3.598 1.586-3.598 2.542 0 .954 1.368 1.913 3.598 2.54.273-.868.603-1.717.985-2.54a20.356 20.356 0 0 1-.985-2.542zm10.572 6.095-.11-.392a19.628 19.628 0 0 0-1.137-2.984l-.085-.177.085-.179c.46-.961.839-1.96 1.137-2.984l.11-.39.395.1c2.935.741 4.617 2 4.617 3.453 0 1.452-1.683 2.711-4.617 3.452l-.395.101zm-.41-3.553c.4.866.733 1.718.987 2.54 2.23-.627 3.599-1.586 3.599-2.54 0-.956-1.368-1.913-3.599-2.542a20.683 20.683 0 0 1-.987 2.542z"
            />
            <path
              d="m6.419 8.695-.11-.39c-.826-2.908-.576-4.991.687-5.717 1.235-.715 3.222.13 5.303 2.265l.284.292-.284.291a19.718 19.718 0 0 0-2.02 2.474l-.113.162-.196.016a19.646 19.646 0 0 0-3.157.509l-.394.098zm1.582-5.529c-.224 0-.422.049-.589.145-.828.477-.974 2.138-.404 4.38.891-.197 1.79-.338 2.696-.417a21.058 21.058 0 0 1 1.713-2.123c-1.303-1.267-2.533-1.985-3.416-1.985zm7.997 16.984c-1.188 0-2.714-.896-4.298-2.522l-.283-.291.283-.29a19.827 19.827 0 0 0 2.021-2.477l.112-.16.194-.019a19.473 19.473 0 0 0 3.158-.507l.395-.1.111.391c.822 2.906.573 4.992-.688 5.718a1.978 1.978 0 0 1-1.005.257zm-3.415-2.82c1.302 1.267 2.533 1.986 3.415 1.986.225 0 .423-.05.589-.145.829-.478.976-2.142.404-4.384-.89.198-1.79.34-2.698.419a20.526 20.526 0 0 1-1.71 2.124z"
            />
            <path
              d="m17.58 8.695-.395-.099a19.477 19.477 0 0 0-3.158-.509l-.194-.017-.112-.162A19.551 19.551 0 0 0 11.7 5.434l-.283-.291.283-.29c2.08-2.134 4.066-2.979 5.303-2.265 1.262.727 1.513 2.81.688 5.717l-.111.39zm-3.287-1.421c.954.085 1.858.228 2.698.417.571-2.242.425-3.903-.404-4.381-.824-.477-2.375.253-4.004 1.841.616.67 1.188 1.378 1.71 2.123zM8.001 20.15a1.983 1.983 0 0 1-1.005-.257c-1.263-.726-1.513-2.811-.688-5.718l.108-.391.395.1c.964.243 2.026.414 3.158.507l.194.019.113.16c.604.878 1.28 1.707 2.02 2.477l.284.29-.284.291c-1.583 1.627-3.109 2.522-4.295 2.522zm-.993-5.362c-.57 2.242-.424 3.906.404 4.384.825.47 2.371-.255 4.005-1.842a21.17 21.17 0 0 1-1.713-2.123 20.692 20.692 0 0 1-2.696-.419z"
            />
            <path
              d="M12 15.313c-.687 0-1.392-.029-2.1-.088l-.196-.017-.113-.162a25.697 25.697 0 0 1-1.126-1.769 26.028 26.028 0 0 1-.971-1.859l-.084-.177.084-.179c.299-.632.622-1.252.971-1.858.347-.596.726-1.192 1.126-1.77l.113-.16.196-.018a25.148 25.148 0 0 1 4.198 0l.194.019.113.16a25.136 25.136 0 0 1 2.1 3.628l.083.179-.083.177a24.742 24.742 0 0 1-2.1 3.628l-.113.162-.194.017c-.706.057-1.412.087-2.098.087zm-1.834-.904c1.235.093 2.433.093 3.667 0a24.469 24.469 0 0 0 1.832-3.168 23.916 23.916 0 0 0-1.832-3.168 23.877 23.877 0 0 0-3.667 0 23.743 23.743 0 0 0-1.832 3.168 24.82 24.82 0 0 0 1.832 3.168z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="37px"
            viewBox="0 0 24 24"
            width="37px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M11.572 0c-.176 0-.31.001-.358.007a19.76 19.76 0 0 1-.364.033C7.443.346 4.25 2.185 2.228 5.012a11.875 11.875 0 0 0-2.119 5.243c-.096.659-.108.854-.108 1.747s.012 1.089.108 1.748c.652 4.506 3.86 8.292 8.209 9.695.779.25 1.6.422 2.534.525.363.04 1.935.04 2.299 0 1.611-.178 2.977-.577 4.323-1.264.207-.106.247-.134.219-.158-.02-.013-.9-1.193-1.955-2.62l-1.919-2.592-2.404-3.558a338.739 338.739 0 0 0-2.422-3.556c-.009-.002-.018 1.579-.023 3.51-.007 3.38-.01 3.515-.052 3.595a.426.426 0 0 1-.206.214c-.075.037-.14.044-.495.044H7.81l-.108-.068a.438.438 0 0 1-.157-.171l-.05-.106.006-4.703.007-4.705.072-.092a.645.645 0 0 1 .174-.143c.096-.047.134-.051.54-.051.478 0 .558.018.682.154.035.038 1.337 1.999 2.895 4.361a10760.433 10760.433 0 0 0 4.735 7.17l1.9 2.879.096-.063a12.317 12.317 0 0 0 2.466-2.163 11.944 11.944 0 0 0 2.824-6.134c.096-.66.108-.854.108-1.748 0-.893-.012-1.088-.108-1.747-.652-4.506-3.859-8.292-8.208-9.695a12.597 12.597 0 0 0-2.499-.523A33.119 33.119 0 0 0 11.573 0zm4.069 7.217c.347 0 .408.005.486.047a.473.473 0 0 1 .237.277c.018.06.023 1.365.018 4.304l-.006 4.218-.744-1.14-.746-1.14v-3.066c0-1.982.01-3.097.023-3.15a.478.478 0 0 1 .233-.296c.096-.05.13-.054.5-.054z"
            />
          </svg>
          <svg
            aria-hidden="true"
            class="c12"
            fill="currentColor"
            focusable="false"
            height="37px"
            viewBox="0 0 24 24"
            width="37px"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M22.251 11.82a3.117 3.117 0 0 0-2.328-3.01L22.911 0H8.104L11.1 8.838a3.116 3.116 0 0 0-2.244 2.988 3.12 3.12 0 0 0 1.313 2.536 8.279 8.279 0 0 1-1.084 1.244 8.14 8.14 0 0 1-2.55 1.647c-.834-.563-1.195-1.556-.869-2.446a3.11 3.11 0 0 0-.91-6.08 3.117 3.117 0 0 0-3.113 3.113c0 .848.347 1.626.903 2.182-.048.097-.097.195-.146.299-.465.959-.993 2.043-1.195 3.259-.403 2.432.257 4.384 1.849 5.489A5.093 5.093 0 0 0 5.999 24c1.827 0 3.682-.917 5.475-1.807 1.279-.632 2.599-1.292 3.898-1.612.48-.118.98-.187 1.508-.264 1.07-.153 2.175-.312 3.168-.89a4.482 4.482 0 0 0 2.182-3.091c.174-.994 0-1.994-.444-2.87.298-.48.465-1.042.465-1.647zm-1.355 0c0 .965-.785 1.75-1.75 1.75a1.753 1.753 0 0 1-1.085-3.126l.007-.007c.056-.042.118-.084.18-.125 0 0 .008 0 .008-.007.028-.014.055-.035.083-.05.007 0 .014-.006.021-.006.028-.014.063-.028.097-.042.035-.014.07-.027.098-.041.007 0 .013-.007.02-.007.028-.007.056-.021.084-.028.007 0 .02-.007.028-.007.034-.007.062-.014.097-.02h.007l.104-.022c.007 0 .02 0 .028-.007.028 0 .055-.007.083-.007h.035c.035 0 .07-.007.111-.007h.09c.028 0 .05 0 .077.007h.014c.055.007.111.014.167.028a1.766 1.766 0 0 1 1.396 1.723zM10.043 1.39h10.93l-2.509 7.4c-.104.02-.208.055-.312.09l-2.64-5.385-2.648 5.35c-.104-.034-.216-.055-.327-.076l-2.494-7.38zm4.968 9.825a3.083 3.083 0 0 0-.938-1.668l1.438-2.904 1.452 2.967c-.43.43-.743.98-.868 1.605H15.01zm-3.481-1.098c.034-.007.062-.014.097-.02h.02c.029-.008.056-.008.084-.015h.028c.028 0 .049-.007.076-.007h.271c.028 0 .049.007.07.007.014 0 .02 0 .035.007.027.007.048.007.076.014.007 0 .014 0 .028.007l.097.02h.007c.028.008.056.015.083.029.007 0 .014.007.028.007.021.007.049.014.07.027.007 0 .014.007.02.007.028.014.056.021.084.035h.007a.374.374 0 0 1 .09.049h.007c.028.014.056.034.084.048.007 0 .007.007.013.007.028.014.05.035.077.049l.007.007c.083.062.16.132.236.201l.007.007a1.747 1.747 0 0 1 .48 1.209 1.752 1.752 0 0 1-3.502 0 1.742 1.742 0 0 1 1.32-1.695zm-6.838-.049c.966 0 1.751.786 1.751 1.751s-.785 1.751-1.75 1.751-1.752-.785-1.752-1.75.786-1.752 1.751-1.752zm16.163 6.025a3.07 3.07 0 0 1-1.508 2.133c-.758.438-1.689.577-2.669.716a17.29 17.29 0 0 0-1.64.291c-1.445.355-2.834 1.05-4.182 1.717-1.724.854-3.35 1.66-4.857 1.66a3.645 3.645 0 0 1-2.154-.688c-1.529-1.056-1.453-3.036-1.272-4.12.167-1.015.632-1.966 1.077-2.877.028-.055.049-.104.077-.16.152.056.312.098.479.126-.264 1.473.486 2.994 1.946 3.745l.264.139.284-.104c1.216-.431 2.342-1.133 3.336-2.071a9.334 9.334 0 0 0 1.445-1.716c.16.027.32.034.48.034a3.117 3.117 0 0 0 3.008-2.327h1.167a3.109 3.109 0 0 0 3.01 2.327c.576 0 1.11-.16 1.57-.43.18.52.236 1.063.139 1.605z"
            />
          </svg>
        </div>
      </div>
    </div>
  </section>
</div>
`;
